src/v/cluster/node_status_rpc_types.h

```diff
@@ -43,7 +43,7 @@ struct node_status_request
 };
 
 struct node_status_reply
-  : serde::envelope<node_status_request, serde::version<0>> {
+  : serde::envelope<node_status_reply, serde::version<0>> {
```

Comment:
it would be preferred if this hunk was squashed into the previous commit that introduced the node_status_reply

Reply:
Yeah. That's what I meant to do in the first place. Fixed.